### PR TITLE
[W5.11][T11-A4]Eka buyung Lienadi

### PR DESCRIPTION
**Pull request by yungyung04**
This is my First enhancement. Now, the address book will automatically 'list' every time 'Add' or 'Delete' command is utilized correctly. The list generated is the updated one, meaning that the newly added person will be shown if the Add command is used. Similarly, the removed person won't be shown in the updated list.

### CODE REVIEW

File: doc/UserGuide.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ Examples:
   Returns Any person having names `Betsy`, `Tim`, or `John`
 
 ### Deleting a person : `delete`
-Deletes the specified person from the address book. Irreversible.<br>
+Deletes the specified person from the address book and shows the updated list of all persons in the address book. Irreversible.<br>
```
Comment (jeffryhartanto) [inline doc/UserGuide.md:70]: Good job for updating the UserGuide.

File: src/seedu/addressbook/commands/DeleteCommand.java [hunk 2]
```diff
@@ -30,8 +33,8 @@ public CommandResult execute() {
         try {
             final ReadOnlyPerson target = getTargetPerson();
             addressBook.removePerson(target);
-            return new CommandResult(String.format(MESSAGE_DELETE_PERSON_SUCCESS, target));
-
+            List<ReadOnlyPerson> allPersons = addressBook.getAllPersons().immutableListView();
+            return new CommandResult(String.format(MESSAGE_DELETE_PERSON_SUCCESS, target), allPersons);
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/commands/DeleteCommand.java:37]: Another way of doing this is by "calling" list command in the Main.java. Refer to https://github.com/nus-cs2103-AY1718S2/addressbook-level2/pull/606/files

File: test/expected.txt [hunk 3]
```diff
@@ -6,10 +6,12 @@
 || Using storage file : addressbook.xml
 || ===================================================
 || Enter command: || [Command entered:  sfdfd]
-|| add: Adds a person to the address book. Contact details can be marked private by prepending 'p' to the prefix.
+|| add: Adds a person to the address book and shows the updated list of all persons in the address book.
+|| Contact details can be marked private by prepending 'p' to the prefix.
```
Comment (jeffryhartanto) [inline test/expected.txt:10]: Good job for updating the IO test.

**Comment by jeffryhartanto** [comment]
@yungyung04 
Some comments added. Please close the PR after reading comments.

File: src/seedu/addressbook/commands/DeleteCommand.java (hunk 2 above)
Comment (yungyung04) [reply src/seedu/addressbook/commands/DeleteCommand.java:37]: Thanks for the suggestion!

**Pull request closed without merging**